akka-persistence-typed/src/main/scala/akka/persistence/typed/state/scaladsl/DurableStateBehavior.scala

```diff
@@ -160,4 +160,9 @@ object DurableStateBehavior {
    */
   def onPersistFailure(backoffStrategy: BackoffSupervisorStrategy): DurableStateBehavior[Command, State]
 
+  /**
+   * Override to define a custom stash capacity per entity.
```

Comment:
Same as above. No Overriding in the scaladsl.
This trait is marked with `@DoNotInherit`.

Reply:
That makes sense. I changed for both.